ghcide.cabal

```
@@ -2,17 +2,17 @@ cabal-version:      1.20
 build-type:         Simple
 category:           Development
 name:               ghcide
-version:            0.2.0
+version:            0.3.0
 license:            Apache-2.0
 license-file:       LICENSE
-author:             Digital Asset
-maintainer:         Digital Asset
+author:             Digital Asset + Ghcide contributors
+maintainer:         Ghcide contributors
 copyright:          Digital Asset 2018-2020
```

```suggestion
copyright:          Digital Asset and Ghcide contributors 2018-2020
```

That worth changing?